cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -202,6 +202,9 @@ endif()
 
 if(BOOST_BUILD_STATIC)
   set(link_opts link=static)
+  if(NOT BOOST_BUILD_DYNAMIC_VSRUNTIME)
```

Comment:
Do we need to protect it with `if(MSVC)`?

Reply:
no not realy, but it's an visual studio feature only. the check is not necessary.

Reply:
ok